config/hubs/cloudbank.cluster.yaml

```diff
@@ -443,3 +443,41 @@ hubs:
                 - christiaan.desmond@sjcc.edu
                 - sanjay.dorairaj@sjcc.edu
               admin_users: *sjcc_users
+  - name: avc
+    domain:
+      - avc.cloudbank.2i2c.cloud
+    template: basehub
+    auth0:
+      connection: google-oauth2
```

Comment:
There was some comment in Slack from @yuvipanda about maybe using Auth password here instead of Google, IIRC.

Reply:
@ericvd-ucb let me know that avc is on google so this should be ok.